doc/Interns/QuickScreenTest/QuickScreenTest/SettingsFacade.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Resolution.h"
+
+//struct DisplaySettings {
```

Comment:
Is this temporarily removed or permanently?  If you need it later, add a //TODO comment to indicate what you will do with it in the future.

Reply:
oops! only temporarily. I'll add a comment